packages/client/src/components/app/forms/InnerForm.svelte

```diff
@@ -409,15 +415,23 @@
   }
 
   const handleScrollToField = ({ field }) => {
-    const fieldId = get(getField(field)).fieldState.fieldId
+    if (!field.fieldState) {
+      field = get(getField(field))
+    }
+    const fieldId = field.fieldState.fieldId
+    const fieldElement = document.getElementById(fieldId)
+    fieldElement.focus({ preventScroll: true })
     const label = document.querySelector(`label[for="${fieldId}"]`)
-    document.getElementById(fieldId).focus({ preventScroll: true })
-    label.scrollIntoView({ behavior: "smooth" })
+    label.style.scrollMargin = "100px"
+    label.scrollIntoView({ behavior: "smooth", block: "nearest" })
   }
 
   // Action context to pass to children
   const actions = [
-    { type: ActionTypes.ValidateForm, callback: formApi.validate },
+    {
+      type: ActionTypes.ValidateForm,
+      callback: formApi.validate,
+    },
```

Comment:
NAB, but we could revert this to being on one line like the other actions (which is what it was originally).